### ISSUE TITLE
A11y: [Choose PIN] PIN input fields lack programmatically associated ...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6391]
- **Checkpoint ID:** 11.1.3.1d/14.1
- **Component:** Choose PIN

### Issue Description
PIN input fields lack programmatically associated labels.

### Recommendation
Provide programmatic labels for each PIN field.

--- 
*Report Date: 25.02.2026*

**Comment by bmarty** [comment]
@gaelledel I guess they want the current box to be highlighted when it has the focus, so the first box on this screen:

<img width="395" height="387" alt="Image" src="https://github.com/user-attachments/assets/f7adf73a-de33-4470-a6f0-42c2bea812c9" />,

The third on this one:

<img width="400" height="419" alt="Image" src="https://github.com/user-attachments/assets/f3bad895-8bd4-47eb-aefd-5988df0d5ce7" />

Can you provide something please?
